.github/workflows/ci.yml

```diff
@@ -129,6 +129,22 @@ jobs:
       - name: Test build
         run: pnpm run test-build
 
+  test-passed:
+    if: (!cancelled() && !failure())
```

Comment:
We cannot get the result of matrix jobs easily (https://github.com/orgs/community/discussions/26639), so I'm using `if: failure()` instead.